### ISSUE TITLE
file editor flashes error when loading

### ISSUE THREAD

**Issue by johrstrom** [OSC/ondemand#1219]
This is a cosmetic bug because it doesn't affect functionality but when one loads the file editor there's a flash in the beginning of the page load that shows the error messages about not being able to find the file.

Apparently this is the default page before the javascript loads.  Now, this is the same as the 1.8 version, it only seems in 1.8 when the page loads, the javascript loads very fast too so. So even though the first page with this error message is rendered, the javascript kicks in very fast and replaces it (hence the performance label as well).

TL;DR: We need to speed up the file editor page load so that the default error that flashes is imperceptible.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201735133575781/1201737419549776) by [Unito](https://www.unito.io)

**Comment by johrstrom** [comment]
This was fixed in #1582.
